github/resource_github_repository.go

```diff
@@ -319,7 +327,6 @@ func resourceGithubRepositoryRead(d *schema.ResourceData, meta interface{}) erro
 	d.Set("svn_url", repo.GetSVNURL())
 	d.Set("git_clone_url", repo.GetGitURL())
 	d.Set("http_clone_url", repo.GetCloneURL())
-	d.Set("archived", repo.GetArchived())
```

Comment:
I might be missing this in the resource but do we set the `archived` attribute in another place? Ahh or we don't need to re-set because it has a default value?

Reply:
Yeah, sorry that looks completely unintentional! Thanks for spotting.